vmm/src/coredump.rs

```diff
@@ -82,12 +88,101 @@ pub struct X86_64ElfPrStatus {
     pub pad3: [u8; 8],
 }
 
+#[repr(C)]
+#[derive(Default, Copy, Clone)]
+pub struct CpuSegment {
+    pub selector: u32,
+    pub limit: u32,
+    pub flags: u32,
+    pub pad: u32,
+    pub base: u64,
+}
+
+pub const DESC_TYPE_SHIFT: u32 = 8;
+pub const DESC_S_SHIFT: u32 = 12;
+pub const DESC_DPL_SHIFT: u32 = 13;
+pub const DESC_P_SHIFT: u32 = 15;
+pub const DESC_P_MASK: u32 = 1 << DESC_P_SHIFT;
+pub const DESC_AVL_SHIFT: u32 = 20;
+pub const DESC_AVL_MASK: u32 = 1 << DESC_AVL_SHIFT;
+pub const DESC_L_SHIFT: u32 = 21;
+pub const DESC_B_SHIFT: u32 = 22;
+pub const DESC_S_MASK: u32 = 1 << DESC_S_SHIFT;
+pub const DESC_G_SHIFT: u32 = 23;
+pub const DESC_G_MASK: u32 = 1 << DESC_G_SHIFT;
```

Comment:
Same here, do you need all these constants to be made public?

Reply:
Done as the same previous comment.